operator/v1/types_csi_cluster_driver.go

```diff
@@ -55,6 +55,7 @@ const (
 	SharedResourcesCSIDriver CSIDriverName = "csi.sharedresource.openshift.io"
 	AlibabaDiskCSIDriver     CSIDriverName = "diskplugin.csi.alibabacloud.com"
 	IBMVPCBlockCSIDriver     CSIDriverName = "vpc.block.csi.ibm.io"
+	NutanixCSIDriver         CSIDriverName = "csi.nutanix.com"
```

Comment:
Are there different types of storage on Nutanix or just the one?

Reply:
Even if différent mode only one CSI and all is managed in storage class